### DISCUSSION TITLE
Virtual Private Servers (VPS) for 100 users

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14770] [General]
I am buying this VPS for 100 users of my company (VPS XL SSD € 26.99).
It is good for YetiForce?

https://contabo.com/en/vps/

**Comment by espe808** [comment]
https://contabo.com/en/vps/
(VPS XL SSD € 26.99)

**Comment by bpabiszczak** [comment]
Yes. You can do PoC [proof of concept]

**Reply by espe808** [reply]
Ok, good.
